purchase_currency/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Set default currency for the company purchase orders and vendor bills.
```

Comment:
```suggestion
Set default currency for the company purchase orders.
```

Would it just be purchase orders? Given that the currency carried itself to vendor bill, I don't think saying we are applying customization to both is valid.